src/odemis/gui/win/acquisition.py

```diff
@@ -47,8 +47,11 @@
 from wx.lib.pubsub import pub
 
 import odemis.gui.model as guimodel
+from odemis.util.filename import guess_pattern, create_filename, update_counter
 
 
+EXTRA_VIEW_STREAMS = [ScannedRemoteTCStream]
```

Comment:
Add a comment about what kind of streams must come here (the ones which will be generated after being folded?).